dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -177,3 +177,27 @@ CWPKI2037E.useraction=Provide at least one non-null, non-empty domain in the con
 CWPKI2038W=CWPKI2038W: The ACME service was unable to revoke the certificate with serial number {0}. The error is ''{1}.''
 CWPKI2038W.explanation=The ACME service attempted to revoke the certificate, but was unable to.
 CWPKI2038W.useraction=Review the error message for details on the failure. The certificate will need to be revoked manually.
+
+CWPKI2039E=CWPKI2039E: The ''{0}'' distinguished name (DN) defined by subjectDN contains a ''{1}'' cn relative distinguished name (RDN) value that does not match any of the defined domains.
+CWPKI2039E.explanation=The cn RDN must match one of the defined domains.
+CWPKI2039E.useraction=Provide a subjectDN attribute value with a cn RDN value that matches one of the defined domains or does not have the cn RDN included.
+
+CWPKI2040E=CWPKI2040E: The cn relative distinguished name (RDN) was not the first RDN in the subjectDN configuration attribute.
+CWPKI2040E.explanation=If the cn RDN is defined, it must be the first RDN in the subjectDN configuration attribute.
+CWPKI2040E.useraction=Provide a subjectDN attribute value that either defines the cn attribute as the first RDN, or does not contain the cn attribute. If the cn attribute is not defined, the first defined domain is used as the cn RDN value.
+
+CWPKI2041E=CWPKI2041E: The ''{0}'' relative distinguished name (RDN) type in the subjectDN configuration attribute is not supported. The following RDN types are supported: cn, o, ou, c, st, l.
+CWPKI2041E.explanation=The subjectDN attribute value contains an RDN that is not supported.
+CWPKI2041E.useraction=Provide a subjectDN attribute value that does not contain unsupported RDN types. The following RDN types are supported: cn, o, ou, c, st, l.
+
+CWPKI2042E=CWPKI2042E: The ''{0}'' subjectDN attribute value is not a valid distinguished name. The error is ''{1}''.
+CWPKI2042E.explanation=The subjectDN attribute must be a valid distinguished name.
+CWPKI2042E.useraction=Provide a valid distinguished name as the subjectDN attribute value.
+
+CWPKI2043E=CWPKI2043E: The ''{0}'' value is not a valid relative distinguished name (RDN). The error is ''{1}''.
+CWPKI2043E.explanation=The ACME service was not able to create an RDN from the specified value.
+CWPKI2043E.useraction=Review the error message for details on the failure.
```

Comment:
Can you give any info about what constitutes a valid RDN? The user action should offer more than just "see error message", which is a given.

Reply:
```CWPKI2043E.useraction=Ensure the the domain included in the value is a valid RDN value.```